com.core447.StreamController.yml

```diff
@@ -0,0 +1,105 @@
+id: com.core447.StreamController
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: launch.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=home
```

Comment:
`home` is broad and there is redundancy here.

Reply:
ok, will change it

Reply:
Can I keep it?  On my machine I get the default cursor, not my custom one, when I remove `filesystem=home` and I think this might cause issues for other ones as well.

Reply:
If you want to keep it drop xdg-documents or drop home and keep other one. home implies access to documents folder.

Reply:
I removed `filesystem=xdg-documents`